yt/frontends/amrvac/data_structures.py

```diff
@@ -245,56 +245,50 @@ def _set_code_unit_attributes(self):
         # First check if overrides have been supplied, if that's the case use those instead.
         # Assume cgs values and let YT handle conversion if supplied in an 'mks' unit system.
         if self.units_override:
-            unit_length = self.units_override.get('length_unit', (1, 'cm'))
-            unit_numberdensity = self.units_override.get('numberdensity_unit', (1, 'cm**-3'))
-            unit_velocity = self.units_override.get('velocity_unit', (0, 'cm*s**-1'))
-            unit_temperature = self.units_override.get('temperature_unit', (1, 'K'))
-            mylog.info('Overriding numberdensity_unit: {:1.0e} {}.'.format(unit_numberdensity[0], unit_numberdensity[1]))
+            length_override = self.units_override.get('length_unit', (1, 'cm'))
+            numberdensity_override = self.units_override.get('numberdensity_unit', (1, 'cm**-3'))
+            velocity_override = self.units_override.get('velocity_unit', (0, 'cm*s**-1'))
+            temperature_override = self.units_override.get('temperature_unit', (1, 'K'))
+            mylog.info('Overriding numberdensity_unit: {:1.0e} {}.'.format(numberdensity_override[0],
```

Comment:
why is this logging entry needed ?

Reply:
It's not needed actually, but yt prints a similar log message when length, temperature and velocity are overridden. To me it seemed like a good idea to also do it for the number density (because that is not considered in `units_override`)

Reply:
alright. In any case this will go through a yt veteran's review so we'll hear what they think